projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.spec.ts

```diff
@@ -162,22 +169,22 @@ describe('BulkActionsDropdownComponent', () => {
         fixture.detectChanges();
       });
 
-      it('should run action on selection', () => {
+      it('should run action on selection', async () => {
         spyOn(component, 'runAction').and.callThrough();
```

Comment:
Not changed in this PR, but I don't think there is a need to spyOn the runAction method in component. callThrough will invoke the actual implementation anyway, and we are not using the spy created by the spyOn in this testcase anywhere

Reply:
It needed `detectChanges()` instead of `callThrough`, fixed, thanks